sirepo/template/genesis.py

```diff
@@ -232,6 +244,16 @@ def _get_col(col_key):
     )
 
 
+def validate_file(file_type, path):
+    if file_type == "io-partfile":
+        if _is_text_file(path):
+            return "The PARTFILE should be a binary file. Use the DISTFILE to import a text file."
+    elif file_type == "io-distfile":
+        if not _is_text_file(path):
+            return "The DISTFILE should be a text file with columns: X PX Y PY T P."
+    return None
```

Comment:
Do we assume that if it doesn't match a known type, it's correct?

Reply:
That's correct - I added input file checking for 2 common cases, which caused confusion in the omega webinar prep.